framework/base/Security.php

```diff
@@ -455,6 +459,15 @@ public function generateRandomKey($length = 32)
 
         $bytes = '';
 
+        if (function_exists('random_bytes')) {
+            try {
+                $bytes .= random_bytes($length);
```

Comment:
Also I find the existing implementation a bit confusing that we append bytes to empty string instead of initializing it inside the ifs.